_docs/dnastack-launch-with.md

```diff
@@ -44,3 +44,5 @@ If not logged into DNAstack, you will be prompted to login. Otherwise or after l
 You will need to pick a version of your workflow to import and a project to import it into.
 Then hit the button to "Import" and continue from within the DNAstack interface to run your workflow. 
 Note that as with the above approach, you will want to double-check that the workflow specifies a runtime environment (docker, cpu, memory, and disks) if you have trouble importing the workflow and that the workflow has not been imported before. 
+
+Additionally, DNAStack does not currently support HTTP or file-based imports.  Importing a workflow with those imports will result in error.
```

Comment:
When you merge to gh-pages, are these changes visible right away?

Asking because we will be changing the button to be disabled in that case, and so Dockstore should prevent you from running into this error.

Should I just update these docs when I make that change?

Reply:
Yup for right away. 

Addressing both comments, I think it makes sense for the tooltip to have a short description and to link to the appropriate document for a full description and possible workarounds in the case of Firecloud.

Reply:
For DNA stack, the button is disabled/hidden only for the version that can't be handled.  But once you go into the DNAStack interface, I don't think there's anything preventing the user from selecting any version.

Reply:
That's true for FC as well. You can pick any version from their UI.